android/src/app/organicmaps/widget/placepage/PlacePageController.java

```diff
@@ -205,8 +205,18 @@ public ArrayList<MenuBottomSheetItem> getMenuBottomSheetItems(String id)
     return items;
   }
 
+  private void setPlacePageInteractions(boolean enabled)
+  {
+    // Prevent place page scrolling when playing the close animation
+    mPlacePageBehavior.setDraggable(enabled);
+    mPlacePage.setNestedScrollingEnabled(enabled);
+    // Prevent user interaction with place page content when closing
+    mPlacePageContainer.setEnabled(enabled);
```

Comment:
Looks like this call is not propagated to children, right?

Reply:
From my testing it seems that calling `mPlacePageContainer.setEnabled` affects its children. At least I was not able to click on buttons from inside the place page after this change.